packages/react/examples/nextjs/package.json

```diff
@@ -8,8 +8,8 @@
     "start": "next start"
   },
   "dependencies": {
-    "@carbon/ibmdotcom-react": "1.2.0-rc.0",
-    "@carbon/ibmdotcom-styles": "1.2.0-rc.0",
+    "@carbon/ibmdotcom-react": "1.3.0-alpha.7",
+    "@carbon/ibmdotcom-styles": "1.3.0-alpha.7",
```

Comment:
I think once we get this in `rc.1`, we should update the dependency to that so when we push full release the dependencies will update automatically by lerna.